### PR TITLE
add DbUserLocation

### PR DESCRIPTION
**Pull request by grinakate**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/925?issue_count=20&issue_position=9&next_issue_id=924&prev_issue_id=926

### CODE REVIEW

File: src/UserService.Models.Db/DbUserLocation.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+
+namespace LT.DigitalOffice.UserService.Models.Db
+{
+  public class DbUserLocation
+  {
+    public const string TableName = "UserLocations";
```
Comment (Cheephot) [inline src/UserService.Models.Db/DbUserLocation.cs:9]: mb UsersLocations?

File: src/UserService.Data.Provider.MsSql.Ef/Migrations/20211115205800_AddUserLocationTable.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+using LT.DigitalOffice.UserService.Models.Db;
+using System;
+
+namespace LT.DigitalOffice.UserService.Data.Provider.MsSql.Ef.Migrations
+{
+  [DbContext(typeof(UserServiceDbContext))]
+  [Migration("20211115205800_AddUserLocationTable")]
+  class AddUserLocationTable : Migration
+  {
+    protected override void Up(MigrationBuilder builder)
```
Comment (Cheephot) [inline src/UserService.Data.Provider.MsSql.Ef/Migrations/20211115205800_AddUserLocationTable.cs:12]: add down method: droptable

**Review by Cheephot — APPROVED** [review]

File: src/UserService.Data.Provider.MsSql.Ef/Migrations/20211115205800_AddUserLocationTable.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+          BusinessHoursFromUtc = table.Column<DateTime>(nullable: true),
+          BusinessHoursToUtc = table.Column<DateTime>(nullable: true),
+          CreatedBy = table.Column<Guid>(nullable: false),
+          CreatedAtUtc = table.Column<DateTime>(nullable: false),
+          ModifiedBy = table.Column<Guid>(nullable: true),
+          ModifiedAtUtc = table.Column<DateTime>(nullable: true)
+        },
+        constraints: table =>
+        {
+          table.PrimaryKey("PK_UserLocationTable", x => x.Id);
```
Comment (BelehovEgor) [inline src/UserService.Data.Provider.MsSql.Ef/Migrations/20211115205800_AddUserLocationTable.cs:31]: we use names like PK_tablename, so you don't needed table here

File: src/UserService.Data.Provider.MsSql.Ef/Migrations/20211115205800_AddUserLocationTable.cs [hunk 4]
```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using LT.DigitalOffice.UserService.Models.Db;
+using System;
+
+namespace LT.DigitalOffice.UserService.Data.Provider.MsSql.Ef.Migrations
+{
+  [DbContext(typeof(UserServiceDbContext))]
+  [Migration("20211115205800_AddUserLocationTable")]
+  class AddUserLocationTable : Migration
```
Comment (BelehovEgor) [inline src/UserService.Data.Provider.MsSql.Ef/Migrations/20211115205800_AddUserLocationTable.cs:10]: public

**Review by BelehovEgor — APPROVED** [review]

File: src/UserService.Models.Db/DbUserLocation.cs (hunk 1 above)
Comment (grinakate) [reply src/UserService.Models.Db/DbUserLocation.cs:9]: ok

File: src/UserService.Data.Provider.MsSql.Ef/Migrations/20211115205800_AddUserLocationTable.cs (hunk 3 above)
Comment (grinakate) [reply src/UserService.Data.Provider.MsSql.Ef/Migrations/20211115205800_AddUserLocationTable.cs:31]: ok

File: src/UserService.Data.Provider.MsSql.Ef/Migrations/20211115205800_AddUserLocationTable.cs (hunk 4 above)
Comment (grinakate) [reply src/UserService.Data.Provider.MsSql.Ef/Migrations/20211115205800_AddUserLocationTable.cs:10]: ok

File: src/UserService.Data.Provider.MsSql.Ef/Migrations/20211115205800_AddUserLocationTable.cs (hunk 2 above)
Comment (grinakate) [reply src/UserService.Data.Provider.MsSql.Ef/Migrations/20211115205800_AddUserLocationTable.cs:12]: ok

**Review by actimellka — APPROVED** [review]

**Review by irlirla — APPROVED** [review]

File: src/UserService.Data.Provider.MsSql.Ef/Migrations/20211115205800_AddUsersLocationsTable.cs [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        },
+        constraints: table =>
+        {
+          table.PrimaryKey("PK_UsersLocations", x => x.Id);
+        });
+    }
+    protected override void Down(MigrationBuilder builder)
+    {
+      builder.DropTable(
+          name: DbUserLocation.TableName);
```
Comment (MariyaSin) [inline src/UserService.Data.Provider.MsSql.Ef/Migrations/20211115205800_AddUsersLocationsTable.cs:37]: fix tab

Comment (grinakate) [reply src/UserService.Data.Provider.MsSql.Ef/Migrations/20211115205800_AddUsersLocationsTable.cs]: done

**Review by MariyaSin — APPROVED** [review]

**Pull request merged**